<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>281</width>
     <height>251</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Глав. окно</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Число</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sval">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>42</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSpinBox" name="fval2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>42</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="sumbtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Сумма</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="fval">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>42</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="updbtn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Обновить</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="sval2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Позиция</string>
     </property>
    </widget>
    <widget class="QPushButton" name="createbtn">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Построить</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Сохр/загр</string>
    </attribute>
    <widget class="QPushButton" name="lbtn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Загрузить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sbtn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="fileline">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Название:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dbtn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
